cpp/cmake_modules/FindThrift.cmake

```diff
@@ -100,7 +100,7 @@ if(Thrift_FOUND OR THRIFT_FOUND)
                                    INTERFACE_INCLUDE_DIRECTORIES "${THRIFT_INCLUDE_DIR}")
   if(WIN32 AND NOT MSVC)
     # We don't need this for Visual C++ because Thrift uses
-    # "#pragma comment(lib, "Ws2_32.lib")" in
+    # "#pragma comment(lib, "ws2_32.lib")" in
```

Comment:
We should not change this because this is the code in Thrift: https://github.com/apache/thrift/blob/master/lib/cpp/src/thrift/windows/config.h#L66

Reply:
That makes sense, I adjusted this.

I should probably double check that the cross compile still works without this, though. Will report once I have tried that.